src/helpers/logs.ts

```diff
@@ -0,0 +1,17 @@
+import { Entry, Log } from '@google-cloud/logging';
+
+const LOG_METADATA = {
+    resource: {
+        type: 'cloud_function',
+        labels: {
+            custom_log_entry: 'true',
+            function_name: 'xplorers-bot',
+            region: 'us-central1'
+        }
+    }
+};
+
+export async function writeLog(log: Log, message: Record<string, unknown>) {
+    const entry: Entry = log.entry(LOG_METADATA, message);
```

Comment:
Adding log severity here should help like `info`, `warning`, `error`.

Reply:
The log severity is defined as part of LOG_METADATA that gets passed into this function